app/src/main/java/de/xikolo/controllers/dialogs/UnsupportedIntentDialog.kt

```diff
@@ -0,0 +1,58 @@
+package de.xikolo.controllers.dialogs
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import com.yatatsu.autobundle.AutoBundleField
+import de.xikolo.R
+import de.xikolo.controllers.dialogs.base.BaseDialogFragment
+
+class UnsupportedIntentDialog : BaseDialogFragment() {
+
+    companion object {
+        @JvmField
+        val TAG: String = UnsupportedIntentDialog::class.java.simpleName
+    }
+
+    @AutoBundleField(required = false)
+    var fileMimeType: String? = null
+
+    var listener: Listener? = null
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val message = getString(R.string.dialog_unsupported_intent_message) +
+            (
+                fileMimeType?.let {
+                    "\n\n${getString(R.string.dialog_unsupported_intent_message_file, it)}"
```

Comment:
Hmm, a bit ugly. Maybe that could be inserted as a format argument into `dialog_unsupported_intent_message` and the two linebreaks could be part of it too?

Reply:
I added it to the beginning of `R.string.dialog_unsupported_intent_message_file`